ruby/Rakefile

```diff
@@ -0,0 +1,151 @@
+# frozen_string_literal: true
+
+require "bundler/gem_tasks"
+require "rspec/core/rake_task"
+require "standard/rake" unless RUBY_VERSION < "2.5"
+
+require "fileutils"
+require "http"
+require "pry"
+require "rubygems/package"
+
+RSpec::Core::RakeTask.new(:spec)
+
+LIB_GITHUB_RELEASES = {
+  "0.2.0" => [
+    {
+      file: "libddprof-x86_64-unknown-linux-gnu.tar.gz",
+      sha256: "cba0f24074d44781d7252b912faff50d330957e84a8f40a172a8138e81001f27",
+      ruby_platform: "x86_64-linux"
+    },
+    {
+      file: "libddprof-x86_64-alpine-linux-musl.tar.gz",
+      sha256: "d519a6241d78260522624b8e79e98502510f11d5d9551f5f80fc1134e95fa146",
+      ruby_platform: "x86_64-linux-musl"
```

Comment:
It's a good question!

Ideally we would upload each `ruby_platform` variant separately to rubygems.org, and Ruby would pick the right one. But [there's a bunch of bugs around that](https://github.com/rubygems/rubygems/issues/3174) and even if the bugs were fixed, we support really old Ruby versions that wouldn't be running the fixed versions.

So actually we're shipping both variants inside the same release on rubygems.org, and then the consumer will pick the right one:

```ruby
def self.pkgconfig_folder
  current_platform = Gem::Platform.local.to_s

  return unless available_binaries.include?(current_platform)

  pkgconfig_file = 
    Dir.glob("#{vendor_directory}/#{current_platform}/**/ddprof_ffi.pc").first # <--- magic happens here
```

But TL;DR I don't think this will confuse customers because they will only get libddprof through dd-trace-rb, which will use the code above to pick the right one and ignore the other.